plugins/woocommerce/changelog/51185-JQUERY_ON_DOCUMENT

```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: tweak
+Comment: Not needed as it doesn't change how woocommerce is working or is used.
+
```

Comment:
```suggestion
Significance: minor
Type: enhancement

Make the scripts that initialize the meta boxes on the Edit Order screen accessible to third parties
```

😕 The CI changelog automation reverted the suggestion I made earlier. I made sure it won't do it again, just going to update this again.